listenstore/listenstore/listen.py

```diff
@@ -21,7 +22,7 @@ def __init__(self, user_id=None, timestamp=None, artist_msid=None, album_msid=No
     def from_json(cls, j):
         """Factory to make Listen() objects from a dict"""
         return cls(  user_id=j['user_id']
-                  , timestamp=j['listened_at']
+                  , timestamp=datetime.utcfromtimestamp(j['listened_at'])
```

Comment:
Commas should be after arguments. This is not the same as SQL, Python supports trailing commas.